Functions/Gherkin.ps1

```diff
@@ -830,7 +830,7 @@ function ConvertTo-HashTableArray {
         if (!${Column Names}) {
             & $SafeCommands["Write-Verbose"] "Reading Names from Header"
```

Comment:
In addition, in [Gherkin.ps1, L409](https://github.com/pester/Pester/blob/cc8319f5f67a7199bdc008bf95f64d02fca73553/Functions/Gherkin.ps1#L409), could it be changed to avoid the error you're experiencing with the unit tests? To something like:

```powershell
if ($Scenario.Contains("Examples")) {
```